src/collect.rs

```diff
@@ -133,6 +135,7 @@ impl Default for ObjectSpace {
     fn default() -> Self {
         let header = new_gc_list();
         Self {
+            collect_on_drop: Cell::new(true),
             list: RefCell::new(header),
```

Comment:
I guess that it is more rational to keep initialization order the same as that of field declaration:
```suggestion
            list: RefCell::new(header),
            collect_on_drop: Cell::new(true),
```